packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -137,11 +137,12 @@ abstract mixin class WidgetsBindingObserver {
   ///   late Size _lastSize;
   ///
   ///   @override
-  ///   void initState() {
-  ///     super.initState();
+  ///   void didChangeDependencies() {
+  ///     super.didChangeDependencies();
   ///     // [View.of] exposes the view from `WidgetsBinding.instance.platformDispatcher.views`
   ///     // into which this widget is drawn.
   ///     _lastSize = View.of(context).physicalSize;
+  ///     WidgetsBinding.instance.removeObserver(this);
   ///     WidgetsBinding.instance.addObserver(this);
```

Comment:
The add should stay in initState, and no need to remove it in didChangeDependencies.

Reply:
Done